app/views/validation.scala.html

```diff
@@ -37,6 +37,12 @@
             </div>
             <div id="upper-menu-holder">
                 <div id="upper-menu-title-bar"></div>
+                <div id="label-control-holder"></div>
```

Comment:
I don't think `label-control-holder` and `label-control-button` are descriptive enough. Maybe something like `label-visibility-control-holder` and `label-visibility-control-button`?

Reply:
Fixed